spec/cose/key/ec2_spec.rb

```diff
@@ -2,9 +2,18 @@
 
 require "cose/key/ec2"
 require "openssl"
+require "base64"
+require "cose/key"
 
 RSpec.describe COSE::Key::EC2 do
   describe ".new" do
+    it "tests" do
+      encoded_key='pQECAyYgASFYHynGYDi87vkqpFOep_onzrmNjPdVBthCuIua9pvBCssiWCBZnNAreTzLOVZrLcTrh6eh-v5GrdemuIS-bVvXrk7Wdw=='
+      cose_key=COSE::Key.deserialize(Base64.urlsafe_decode64(encoded_key))
+      cose_key.to_pkey
+    end
+
+
```

Comment:
I would just remove this test to be honest, I think we have enough tests for the `to_pkey` method and the one you added that covers the issue at hand is enough to me 🙂

Reply:
Honestly I think this was a snippet I used for debugging and forgot to delete :). Will remove and send a new commit later.

Reply:
Done